hybrasyl/Map.cs

```diff
@@ -442,8 +441,11 @@ public void Insert(VisibleObject obj, byte x, byte y, bool updateClient = true)
 
                     foreach (var target in affectedObjects)
                     {
-                        target.AoiEntry(obj);
-                        obj.AoiEntry(target);
+                        if (target is User usr)
+                        {
+                            usr.AoiEntry(obj);
+                            usr.AoiEntry(target);

```

Comment:
I'm not sure I understand this change, at all. The target needs to know about the inserted object, and vice versa?

Reply:
this has been reverted. was leftover from testing invis mob issue.